gwt-regexp-j2cl-tests/pom.xml

```diff
@@ -108,19 +78,9 @@
     </build>
 
     <repositories>
-        <repository>
-            <id>${vertispan.j2cl.repo.id}</id>
-            <name>${vertispan.j2cl.repo.name}</name>
-            <url>${vertispan.j2cl.repo.url}</url>
```

Comment:
same url was listed three times with different ID and name, doesnt serve a purpose. just need it once in <repositories> and once in <pluginRepositories>